src/lib/Canvas.svelte

```diff
@@ -114,6 +114,7 @@
     <slot />
   {/if}
 </canvas>
+<svelte:window on:scroll={(e) => onScroll(getCtx(), getRootCtx(), e)} />
 
```

Comment:
clever! Although I was not able to move the cursor onto an interactive object by scrolling and affecting it by that. Only when I moved the mouse a bit.

Reply:
I then realized that this behaviour is the same as it was before, so that's fine.

Reply:
Hmm I guess the scroll event doesn't contain a new pointerposition we cant do a call to setPointerFromEvent(). Since it's not working, I will remove all scroll related stuff.